action.yml

```diff
@@ -1,12 +1,12 @@
-name: 'Notify for pending release'
+name: 'Notification for pending release'
 description: 'Create an issue if there is a release pending'
 inputs:
   github-token:
     description: 'GitHub token'
     required: true
-  days-to-stale-release:
-    description: 'The number of days after which notification should trigger after last unreleased commit'
-    required: true
+  days-to-ignore:
+    description: 'Unreleased commits older than the no. of days mentioned in this setting shall be reported'
```

Comment:
this description is not accurate because, as we discussed, because we're opening an issue if there are commits stale-days after the last release and we include all of them in the issue. the previous one was slightly better but still not completely accurate either